tests/compile/test_ops.py

```diff
@@ -32,10 +33,11 @@ def cumprod(x):
         assert np.allclose(r, r0), (r, r0)
 
     def test_2arg(self):
-        x = dmatrix("x")
-        x.tag.test_value = np.zeros((2, 2))
-        y = dvector("y")
-        y.tag.test_value = [0, 0, 0, 0]
+        with pytest.warns(FutureWarning):
```

Comment:
For tests that are not specific about test_values we should just stop using the machinery